entrypoint.sh

```diff
@@ -31,5 +32,11 @@ if [ "$destroy_volumes" = "true" ]; then
   volume_flag="-v"
 fi
 
-echo running: okteto pipeline destroy $log_level --name "${name}" ${params} $volume_flag --wait
-okteto pipeline destroy $log_level --name "${name}" ${params} $volume_flag --wait
+if [ "$dependencies" = "true" ]; then
+  dependencies_flag="--dependencies"
+else
+  dependencies_flag="--dependencies=false"
+fi
```

Comment:
I would follow the same pattern we have for https://github.com/okteto/pipeline/pull/26.

We should probably do the distinction here about default value used vs explicitly set `false`. This is specially relevant given that the flag might change at instance level, and you might want to force a value. If the default value is used (`false`), and the instance forces the dependencies, it should delete the dependency. But, if I set explicitly the false value, and the instance force the destroy, It should not delete the dependencies